### PR TITLE
RMMIS-6563: Backspace on masked fields

### PR DESCRIPTION
**Pull request by traviscooperscdhhs**
e.preventDefault() is needed, but also stops event bubbling.  Created new event and bubbled up.

Removed test, getting error on `new Event` constructor in test, so not sure if we need a polyfill, or something else. 

Wanted to get to DEV so we can test asap.   Please advice if you know how to test.

### CODE REVIEW

**Comment by traviscooperscdhhs** [comment]
So tested locally, and it works.  Just not sure about test. Didn't want to spend lots of time on it yet.  Wanted to get fix out ASAP.

File: src/Field.jsx [hunk 1]
```diff
@@ -176,13 +176,16 @@ class Field extends React.Component {
     if (this.props.mask && e.keyCode === BACKSPACE) {
       let value = this.props.value.slice(0, -1);
       e.preventDefault();
```
Comment (kjschulz) [inline src/Field.jsx]: I know I'm being picky, but can we move `e.preventDefault();` either above or below the var declarations to make it MORE clean?

**Comment by kjschulz** [comment]
@traviscooper, I think that's correct.  Will fail because phantomJS doesn't support the new Event() syntax even thought the other way is deprecated :+1:

**Comment by kjschulz** [comment]
:+1: :+1:

**Comment by traviscooperscdhhs** [comment]
Updated

**Pull request merged**